lib/node_modules/@stdlib/repl/lib/main.js

```diff
@@ -158,6 +159,7 @@ function REPL( options ) {
 	setNonEnumerableReadOnly( this, '_isTTY', opts.isTTY );
 	setNonEnumerableReadOnly( this, '_sandbox', opts.sandbox );
 	setNonEnumerable( this, '_quiet', opts.quiet ); // allow this to be internally toggled
+	setNonEnumerable( this, '_autoClose', opts.autoClose );
```

Comment:
We should have a private `settings` object for storing these kinds of option values.